package.json

```diff
@@ -39,15 +39,15 @@
     "yarn": "^1.4.2"
   },
   "scripts": {
-    "build": "lerna run build",
+    "build": "cross-env SKIP_PREFLIGHT_CHECK=true lerna run build",
```

Comment:
why is this needed?

Reply:
CRA2 requires a version of eslint that is more recent than the version installed by our dependencies (semistandard); this environment variable suppresses the warning